src/service-worker.ts

```diff
@@ -110,8 +111,10 @@ export async function generateServiceWorker(options: AddServiceWorkerOptions):
     return path.join(buildRoot, filePath);
   });
 
-  // swPrecache will determine the right urls by stripping buildRoot
-  swPrecacheConfig.stripPrefix = buildRoot;
+  // swPrecache will determine the right urls by stripping buildRoot.
+  // NOTE:(usergenic) sw-precache generate() apparently to replaces the
```

Comment:
Also wat??? That's crazy. Can you also file a bug with them to get this fixed? (At the very least, this means that some of their README examples don't actually work on Windows).